### PR TITLE
Add Values for Distance and Distance Lines

### PR DESCRIPTION
**Pull request by arne182**
On the Toyta display the button and the lines that are displayed can be read with these can messages.

### CODE REVIEW

**Comment by energee** [comment]
This is a generated file so you would want to make this change in _toyota.dbc and run generator.py.

**Comment by rbiasini** [comment]
Also, the change should be done in opendbc repo first.

File: opendbc/toyota_rav4_hybrid_2017_pt_generated.dbc [hunk 1]
```diff
@@ -128,6 +128,7 @@ BO_ 742 LEAD_INFO: 8 DSU
 
 BO_ 835 ACC_CONTROL: 8 DSU
  SG_ ACCEL_CMD : 7|16@0- (0.001,0) [-20|20] "m/s2" HCU
+ SG_ ACC_DISTANCE : 20|1@0+ (1,0) [0|1] "" XXX
```
Comment (rbiasini) [inline opendbc/toyota_rav4_hybrid_2017_pt_generated.dbc]: @arne182 what is this signal exactly?

File: opendbc/toyota_rav4_hybrid_2017_pt_generated.dbc [hunk 2]
```diff
@@ -128,6 +128,7 @@ BO_ 742 LEAD_INFO: 8 DSU
...
  SG_ SET_ME_X63 : 23|8@0+ (1,0) [0|255] "" HCU
```
Comment (rbiasini) [inline opendbc/toyota_rav4_hybrid_2017_pt_generated.dbc]: also, you need to split this signal, which includes the one ^^

**Comment by rbiasini** [comment]
closing since this should be done in opendbc repo

**Comment by arne182** [comment]
This is the Toyota Stock distance button press and responce.

**Comment by arne182** [comment]
How would you suggest splitting it?

**Pull request closed without merging**